core/node/events/stream.go

```diff
@@ -734,7 +736,16 @@ func (s *streamImpl) addEventLocked(ctx context.Context, event *ParsedEvent) err
 	// TODO: for some classes of errors, it's not clear if event was added or not
 	// for those, perhaps entire Stream structure should be scrapped and reloaded
 	if err != nil {
-		return err
+		var sb strings.Builder
```

Comment:
If the minipool is large, this line could perhaps have performance implications, I'm not sure how it would affect connect if we returned an error with >> metadata over the wire.